onyo/lib/commands.py

```diff
@@ -171,6 +173,108 @@ def onyo_config(inventory: Inventory,
                                             'config: modify repository config')
 
 
+def _edit_asset(inventory: Inventory, asset: dict, operation: Callable, editor: Optional[str]) -> dict:
```

Comment:
FTR: I'd still only do that, when there's a couple of them. Say: 2 (or 3 if there's `self`) arguments - one line. Otherwise one argument per line. However, obviously still valid in this case. Will do.